packages/server/database/migrations/20210317155500-swapTatorIdWithCreatedBy.ts

```diff
@@ -0,0 +1,33 @@
+import {R} from 'rethinkdb-ts'
+export const up = async function(r: R) {
+  const now = new Date()
+  try {
+    await r
```

Comment:
we can combine both of these queries into 1 by using `replace, merge, without`. try it out!